### PR TITLE
CLOUD-54118 fixed validation on stack parameters

### PR DESCRIPTION
**Pull request by doktoric**
@keyki @mhmxs @biharitomi @schfeca75 

Stackvalidation using our reactor framework now.

at least 3 LGTM needs for merge.

### CODE REVIEW

**Comment by biharitomi** [comment]
LGTM

File: core/src/main/java/com/sequenceiq/cloudbreak/controller/StackController.java [hunk 1]
```diff
@@ -156,7 +161,7 @@ public Response put(Long id, UpdateStackJson updateRequest) {
         CbUser user = authenticatedUserService.getCbUser();
         MDCBuilder.buildUserMdcContext(user);
         Stack stack = stackService.getById(id);
-        if ("BYOS".equals(stack.cloudPlatform())) {
+        if (CloudConstants.BYOS.equals(stack.cloudPlatform())) {
```
Comment (keyki) [inline core/src/main/java/com/sequenceiq/cloudbreak/controller/StackController.java]: Isn't is supposed to be BYOS.name() as we're comparing it with a string,

Comment (doktoric) [inline core/src/main/java/com/sequenceiq/cloudbreak/controller/StackController.java]: CloudConstants.BYOS is a string

Comment (keyki) [inline core/src/main/java/com/sequenceiq/cloudbreak/controller/StackController.java]: Oh, thought it's an enum.

**Comment by keyki** [comment]
LGTM

**Comment by mhmxs** [comment]
LGTM

**Pull request merged**